test/clj/rems/api/test_forms.clj

```diff
@@ -397,29 +411,26 @@
                                 :field/privacy :public}]}]
     (testing "creating"
       (testing "invalid request"
-        (letfn [(fail-request [command]
-                  (let [response (-> (request :post "/api/forms/create")
-                                     (authenticate api-key user-id)
-                                     (json-body command)
-                                     handler)]
-                    (or (= 400 (:status response))
-                        (not (get-in response [:body :success])))))]
```

Comment:
The bug was here: `:body` needs to be read & parsed. It's a stream and thus never contains `:success`.